src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -534,9 +534,9 @@ public override IntermediateNode Visit(DottedNameNode node, IRTranslatorContext
 
                     if (DType.Color.Accepts(resultType))
                     {
-                        Contracts.Assert(value is double);
-                        result = new ColorLiteralNode(context.GetIRContext(node), Convert.ToUInt32((double)value));
-                    }
+                        var uintValue = Convert.ToUInt32(value);
+                        result = new ColorLiteralNode(context.GetIRContext(node), System.Drawing.Color.FromArgb((byte)((uintValue >> 24) & 0xFF), (byte)((uintValue >> 16) & 0xFF), (byte)((uintValue >> 8) & 0xFF), (byte)(uintValue & 0xFF)));
```

Comment:
nit - can we put this System.Drawing.Color.FromArgb(...) in its own helper method